### DISCUSSION TITLE
It's trying to delete "HoYoKProtect.sys" Is this safe?

### DISCUSSION THREAD

**Discussion by mxlfur** [34736384/genshin-fps-unlock#318] [General]
Is this really safe for this file to just delete it? I mean the games still runs in unlocked frames so it doesn't really bothered me that much, but I'm just curious why is this new update trying to delete a file now. I've had been using this frame unlocker for quite long now and this is the first time my anti-virus noticing it.

**Comment by raygiizs** [comment]


**Reply by mxlfur** [reply]
I was never talking about the game, it's just if that file is deleted is it still safe for the game? I also heard it's their own anti-cheat system, so kinda a bit concerning now.
Author might as well remove that execution on deleting that file because it still works without that deletion at my end. 
_**Or is it unsafe now for unlocking FPS because of that file that's why it's need to be deleted?**_

**Comment by 34736384** [comment]
The unlocker does not delete any files

**Reply by mxlfur** [reply]
![image_2023-12-27_060201104](https://github.com/34736384/genshin-fps-unlock/assets/154902176/bc8d614a-e470-47fa-8a7d-ebc825734205)

**Reply by 34736384** [reply]
No it doesn't 
![image](https://github.com/34736384/genshin-fps-unlock/assets/25654009/377c8f6e-a70b-43c6-a6e9-1442e096c3c6)

**Reply by mxlfur** [reply]
Funny thing. I just checked "Auto Close" and I got no more notification of my anti-virus of that thing anymore. e_O

_I don't know what just happened_

It did really kept showing up before I did that
